src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -59,6 +61,51 @@ const AuthScreen = () => {
     }
   };
 
+  const getAuthStatus = async () => {
+    setLoading(true);
+    const deviceInfo = await DeviceInfo.getDeviceName();
+    const deviceId = await DeviceInfo.getUniqueId();
+
+    try {
+      const data = await fetch(AuthApis.QR_AUTH_API, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          device_info: deviceInfo,
+          user_id: 'BE9a4sGXFLDwxZU3DSiq', //TODO: replace with scanner results
+          device_id: deviceId,
+        }),
+      });
+
+      if (data.ok) {
+        console.log('patch call successfull');
```

Comment:
Why log?

Reply:
it is placeholder for the actual functionality